src/lang/en.ts

```diff
@@ -85,5 +85,8 @@ export default {
     },
     keyTip: 'You can edit and input any value here,and press Enter to confirm.',
     hashOnTip: 'Select a hash on.'
+  },
+  message: {
+    cannotAddMore: 'You cannot add more item!'
```

Comment:
`more items`

Reply:
I made some changes, but I don't know if that's what you meant.

Reply:
We would use `items` here, because `more` means many items :)

Reply:
such as `cannotAddMoreItems` ?